### ISSUE TITLE
Create a script(s) to update "Conferences" and "Meetups" sections README

### ISSUE THREAD

**Issue by obahareth** [obahareth/awesome-mena-conferences-and-meetups#8]
This script should read from `data/conferences/en.json` and `data/meetups/en.json` and update a "Conferences" and "Meetups" section in the README, without affecting other sections in the README.

The idea is that this script would later be run automatically by CI and always keeps the README up to date, so that this repo can join the [awesome lists](https://github.com/sindresorhus/awesome) (having everything in the README is a requirement).

**Comment by lumie31** [comment]
The file `en.yml` does not exist

**Comment by obahareth** [comment]
You're absolutely correct @lumie31, I wrote this when I was thinking of using YAML, it's `en.json` now. I'm sorry for the confusion.

**Comment by lumie31** [comment]
> You're absolutely correct @lumie31, I wrote this when I was thinking of using YAML, it's `en.json` now. I'm sorry for the confusion.

Ooh great
